### DISCUSSION TITLE
taro每次编译小程序都生成了一个.swc文件

### DISCUSSION THREAD

**Discussion by hanxiao19990114** [NervJS/taro#13917] [Q & A]
![image](https://github.com/NervJS/taro/assets/100828528/27acef1c-dcf2-43bb-b202-37a6d62fbada)

**Comment by hanxiao19990114** [comment]
请问这个是什么， 我编译完后把它删掉也没啥影响 如何能让他不生成这个文件

**Reply by yoyo837** [reply]
这是个临时目录，可以把整个 .swc 文件夹添加到git ignore

**Reply by hanxiao19990114** [reply]
能不能让他不生成呢

**Reply by yoyo837** [reply]
![image](https://github.com/NervJS/taro/assets/6134547/ccd5e0e9-2878-4f65-b9da-4075263a1fb1)

不能

**Reply by hanxiao19990114** [reply]
好的 感谢
